recipe/meta.yaml

```
@@ -1,43 +1,109 @@
-{% set version = "2.4.1" %}
+{% set name = "esda" %}
+{% set version = "2.5.1" %}
 
 package:
-  name: esda
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/e/esda/esda-{{ version }}.tar.gz
-  sha256: 308b43cdc2f7af9b01c4a71fb1869092d48bcdf953e99dba8fc8239745779f1e
+  url: https://github.com/pysal/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 8557fd5da1d7ee23cdcc8cb8f83ab8e772f7200b647303f3b0bb8836ae19fe9e
+  patches:
+    - patches/0001-tests-must-use-absolute-path.patch
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # Need to avoid error: setuptools-scm was unable to detect version !!!
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  # scipy >=1.9,<1.12 is not avaliable on py313
+  # libpysal is not avaliable for py39
+  skip: True  # [py<310 or py>312]
+  # s390x skipped: geopandas, fiona, shapely, and numba is not avaliable and we can`t test is it works at all.
+  skip: True  # [s390x]
 
 requirements:
+  build:
+    - patch  # [not win]
+    - m2-patch  # [win]
   host:
     - python
+    - wheel
+    - setuptools >=61.0
+    - setuptools_scm >=6.2
     - pip
   run:
-    - python >3.4
-    - libpysal
-    - pandas
-    - scikit-learn
-    - scipy >=0.11
+    - python
+    - libpysal >=4.10
```

could you please try (together with the other suggestions):
```suggestion
    - libpysal
    - libpysal >=4.10  # [py>39]
```